R/process_data_set_jordan.R

```diff
@@ -0,0 +1,33 @@
+library(tidyverse)
+library(usethis)
+library(stringr)
+library(lubridate)
+library(janitor)
+
+# Import dataset from csv
+fight_songs <- read_csv("data-raw/fight-songs/fight-songs.csv")
+
+names(fight_songs)[12] <- "num_fights"
```

Comment:
Hard coding with a `12` can be dangerous, especially if future versions of this data have a different order to the variables. Could you use `rename()` instead, and then connect pipe with `mutate()` below so that we have a single chain?